core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1001,6 +1001,20 @@ write_files:
              | base64 -d > $f
              mv -f $f ${encKey%.enc}
            done;
+
+           {{- if .AssetsConfig.HasAnyAuthTokens }}
+           authDir=/etc/kubernetes/auth
+           echo generating $authDir/computed-tokens.csv
+           echo > $authDir/computed-tokens.csv
+
+           {{- if .AssetsConfig.HasTLSBootstrapToken }}
+           echo $(cat $authDir/tls-bootstrap.token),kubelet-bootstrap,10001,system:kubelet-bootstrap >> $authDir/computed-tokens.csv
+           {{- end }}
+           {{- if .AssetsConfig.HasAuthTokens }}
+           cat $authDir/tokens.csv >> $authDir/computed-tokens.csv
```

Comment:
nit but my preference is first write a work file like `tokens.csv` to `/var/run/coreos/tokens.csv` and keep the final one named `$authDir/tokens.csv` rather than `computed-tokens.csv`
so that someone peeked into `$authDir` finds only `tokens.csv` and don't get confused like `which one is the "final" tokens file passed to apiserver, computed-tokens.csv or tokens.csv?`